.flake8

```diff
@@ -1,24 +1,21 @@
 [flake8]
-max-line-length = 88
-ignore = E501, E203, W503
+min_python_version = 3.6.0
+ignore =
+    # E501: Line too long
+    E501,
+    # W503: Line break occurred before a binary operator
+    W503
```

Comment:
This one is also likely to stay @abn -- the accepted practice is opposite what flake8 suggests.

See:
https://www.flake8rules.com/rules/W503.html
https://github.com/psf/black/issues/52
https://github.com/psf/black/pull/36

Although, oddly current flake8 is supposed to have this off by default?
https://github.com/psf/black/blob/6559bdbd9dcf71ba0017d66bb133c744bd83b0e3/docs/guides/using_black_with_other_tools.md
That doesn't appear to be happening here, which is odd.

Reply:
@neersighted FYI the check is off because `ignore` is used and it overrides all on-by-default ignores. There is `extend-ignore` if you want to extend, not override.

Reply:
Yeah, I figured that out. It's actually resolved in master now. Thanks for looking into this!